### ISSUE TITLE
Missing subunit of land register in parcel search response

### ISSUE THREAD

**Issue by edigonzales** [geoadmin/mf-chsdi3#3384]
https://api3.geo.admin.ch/rest/services/api/SearchServer?sr=2056&limit=15&type=locations&origins=address,parcel&searchText=messen%20168

Same parcel number and same municipality. They belong to a different subunit of land register. Without this information one cannot distinguish them.

![Screenshot 2020-02-08 at 19 58 59](https://user-images.githubusercontent.com/2767800/74090598-7fac0900-4aad-11ea-8d53-7dee58b1e36a.png)

**Comment by davidoesch** [comment]
@beat-github  what is the expected result? works as defined https://api3.geo.admin.ch/services/sdiservices.html#search

-> @edigonzales : proposal?

**Comment by edigonzales** [comment]
Es müsste zusätzlich der Namen des Grundbuch(-kreises) in der Antwort stehen resp. im Suchdienst muss das Attribut "subunitoflandregister" drin sein. Bei fusionierten politischen Gemeinden _ohne_ fusioniertes Grundbuch wird es für den Anwender unmöglich das Grundstück zu finden.

Etwa so:

![Screenshot 2020-02-09 at 15 51 46](https://user-images.githubusercontent.com/2767800/74104409-a7a27780-4b54-11ea-8612-64df0cf2a5f1.png)

Kann noch verwirrender werden:

![Screenshot 2020-02-09 at 15 54 49](https://user-images.githubusercontent.com/2767800/74104416-ba1cb100-4b54-11ea-9a1f-e9f818f97838.png)

**Comment by p1d1d1** [comment]
@edigonzales in which model is the attribute "subunitoflandregister" available?

**Comment by edigonzales** [comment]
None that I'm aware of. One could use "nbident" (more or less). But the names of the subunits of land register would still be missing. 

+1 for a fast and pragmatic solution.

**Comment by p1d1d1** [comment]
so instead of having:

Messen 168 (CH 8073 0658 3219) Parzelle
Messen 168 (CH 1818 3206 7404) Parzelle 

we would have:

Messen 168 SO0200002457 (CH 8073 0658 3219) Parzelle
Messen 168 SO0200002443 (CH 1818 3206 7404) Parzelle

which doesn't bring too much IMHO. We already have the EGRID to distinguish between the 2 parcels

**Comment by edigonzales** [comment]
No. What I meant with "fast + pragmatic" is that it can be done without a minimales Geodatenmodell including all the processes (FIG etc.). But with a simple INTERLIS model managed by V+D.

**Comment by edigonzales** [comment]
Wishlist: Distinguish also the parcel type. Parzelle can be "Liegenschaft" or "selbständiges dauerndes Recht", e.g. Baurecht.
